src/electricitybot/electricitybot.py

```diff
@@ -50,7 +51,8 @@ def check_and_send(self):
         current_e_state = self.check_electricity()
         if self.previous_e_state != current_e_state:
             message = self.build_message(current_e_state)
-            self.tg_bot.send_message(chat_id=self.chat_id, text=message)
+            kwargs = dict(chat_id=self.chat_id, message_thread_id=self.thread_id, text=message)
+            self.tg_bot.send_message(**{k: v for k, v in kwargs.items() if v is not None})
```

Comment:
Do not need to overcomplicate it `message_thread_id` parameter have default value `None` so we can just prepare it properly.

```suggestion
            self.tg_bot.send_message(chat_id=self.chat_id, message_thread_id=self.thread_id, text=message)
```